MATLAB.gitlab-ci.yml

```diff
@@ -40,7 +41,7 @@
 #
 command:
   extends: .matlab_defaults
-  script: matlab -batch mycommand
+  script: matlab -batch "mycommand"
```

Comment:
On lines 40 and 48, replace `mycommand` with `"mycommand"`

Reply:
I think we probably want to leave this as `mycommand` right? Because we don't want them to substitute the quotes, just the token inside the quotes.

Reply:
Yes, you can just use `mycommand` and `mytask` everywhere in the text.